schoolreg_api/settings.py

```diff
@@ -38,7 +38,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'rest_framework'
+    'rest_framework',
+    'API_register'
```

Comment:
Keeping comma at the end in style like this helps to avoid conflicts working with multiple devs. Anyway it's rather matter of codebase style guide so team might create different rules